VMS/Pages/Homepage/SloganAndVideo.razor

```diff
@@ -1,12 +1,25 @@
 ﻿@inject IJSRuntime JsRuntime
 
-<div class="my-slogan">
-    <p class="text-center h-100 m-0 py-2">Go Volunteer: Cong dong Tinh nguyen truc tuyen Truong Dai hoc Kinh te TP.HCM</p>
+<div class="video-header">
+    <div class="video-header__note">
+        <div class="video-header__note__text">
+            GoVirlunteer: Cộng đồng Tình nguyện trực tuyến Trường Đại học Kinh tế TP. HCM
+        </div>
+    </div>
+    <video src=@videoUrl class="video-header__source" loop muted autoplay></video>
+    <div class="video-header__title">
```

Comment:
BA confirmed that 2 lines "Get Ready UEH" and " Go Virlunteer" are belong to the video, we don't have to define them in code

Reply:
You replaced the <video class=".my-homepage-video"> by <video class="video-header__source"> but ".my-homepage-video" still exists in index.js, that causes an error in console.
Another point is Mr.Long said you should use Javascript to show to video instead of just video tag in HTML.
Please take a look at these points

Reply:
i solved it.